### PR TITLE
Python API: Do not rename enumerators.

### PR DESCRIPTION
**Pull request by aniemetz**

### CODE REVIEW

File: examples/api/python/bitvectors_and_arrays.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
     slv = cvc5.Solver()
     slv.setOption("produce-models", "true")
     slv.setOption("output-language", "smtlib")
-    slv.setLogic("QF_AUFBV")
+    slv.setLogic("QF_AUFBITVECTOR_")
```
Comment (mpreiner) [inline examples/api/python/bitvectors_and_arrays.py:28]:
```suggestion
    slv.setLogic("QF_AUFBV")
```

File: examples/api/python/bitvectors.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
 if __name__ == "__main__":
     slv = cvc5.Solver()
-    slv.setLogic("QF_BV") # Set the logic
+    slv.setLogic("QF_BITVECTOR_") # Set the logic
```
Comment (mpreiner) [inline examples/api/python/bitvectors.py:24]:
```suggestion
    slv.setLogic("QF_BV") # Set the logic
```

File: examples/api/python/extract.py [hunk 3]
```diff
@@ -20,29 +20,29 @@
 
 if __name__ == "__main__":
     slv = Solver()
-    slv.setLogic("QF_BV")
+    slv.setLogic("QF_BITVECTOR_")
```
Comment (mpreiner) [inline examples/api/python/extract.py:23]:
```suggestion
    slv.setLogic("QF_BV")
```

File: examples/api/python/floating_point.py [hunk 4]
```diff
@@ -17,13 +17,13 @@
...
 import cvc5
-from cvc5 import Kind, RoundingMode
+from cvc5 import Kind. RoundingMode
 
 if __name__ == "__main__":
     slv = cvc5.Solver()
 
     slv.setOption("produce-models", "true")
-    slv.setLogic("QF_FP")
+    slv.setLogic("QF_FLOATINGPOINT_")
```
Comment (mpreiner) [inline examples/api/python/floating_point.py:26]:
```suggestion
    slv.setLogic("QF_FP")
```

File: examples/api/python/floating_point.py [hunk 5]
```diff
@@ -37,21 +37,28 @@
...
     print("Model for x:", slv.getValue(x))
     print("Model for y:", slv.getValue(y))
 
     # disallow NaNs and Infinities
-    slv.assertFormula(slv.mkTerm(Kind.Not, slv.mkTerm(Kind.FPIsNan, x)))
-    slv.assertFormula(slv.mkTerm(Kind.Not, slv.mkTerm(Kind.FPIsInf, x)))
-    slv.assertFormula(slv.mkTerm(Kind.Not, slv.mkTerm(Kind.FPIsNan, y)))
-    slv.assertFormula(slv.mkTerm(Kind.Not, slv.mkTerm(Kind.FPIsInf, y)))
+    slv.assertFormula(slv.mkTerm(
+        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_ISNAN, x)))
```
Comment (mpreiner) [inline examples/api/python/floating_point.py:55]:
```suggestion
        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_IS_NAN, x)))
```

File: examples/api/python/floating_point.py [hunk 6]
```diff
@@ -37,21 +37,28 @@
...
+    slv.assertFormula(slv.mkTerm(
+        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_ISINF, x)))
```
Comment (mpreiner) [inline examples/api/python/floating_point.py:57]:
```suggestion
        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_IS_INF, x)))
```

File: examples/api/python/floating_point.py [hunk 7]
```diff
@@ -37,21 +37,28 @@
...
+    slv.assertFormula(slv.mkTerm(
+        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_ISNAN, y)))
```
Comment (mpreiner) [inline examples/api/python/floating_point.py:59]:
```suggestion
        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_IS_NAN, y)))
```

File: examples/api/python/floating_point.py [hunk 8]
```diff
@@ -37,21 +37,28 @@
...
+    slv.assertFormula(slv.mkTerm(
+        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_ISINF, y)))
```
Comment (mpreiner) [inline examples/api/python/floating_point.py:61]:
```suggestion
        Kind.NOT, slv.mkTerm(Kind.FLOATINGPOINT_IS_INF, y)))
```

File: examples/api/python/floating_point.py [hunk 9]
```diff
@@ -17,13 +17,13 @@
 ##
 
 import cvc5
-from cvc5 import Kind, RoundingMode
+from cvc5 import Kind. RoundingMode
```
Comment (mpreiner) [inline examples/api/python/floating_point.py:20]:
```suggestion
from cvc5 import Kind, RoundingMode
```

File: examples/api/python/sets.py [hunk 10]
```diff
@@ -39,44 +39,44 @@
...
     two = slv.mkInteger(2)
     three = slv.mkInteger(3)
 
-    singleton_one = slv.mkTerm(Kind.SetSingleton, one)
-    singleton_two = slv.mkTerm(Kind.SetSingleton, two)
-    singleton_three = slv.mkTerm(Kind.SetSingleton, three)
-    one_two = slv.mkTerm(Kind.SetUnion, singleton_one, singleton_two)
-    two_three = slv.mkTerm(Kind.SetUnion, singleton_two, singleton_three)
-    intersection = slv.mkTerm(Kind.SetInter, one_two, two_three)
+    singleton_one = slv.mkTerm(Kind.SET_SINGLEton, one)
```
Comment (mpreiner) [inline examples/api/python/sets.py:70]:
```suggestion
    singleton_one = slv.mkTerm(Kind.SET_SINGLETON, one)
```

File: examples/api/python/sets.py [hunk 11]
```diff
@@ -39,44 +39,44 @@
...
+    singleton_two = slv.mkTerm(Kind.SET_SINGLEton, two)
```
Comment (mpreiner) [inline examples/api/python/sets.py:71]:
```suggestion
    singleton_two = slv.mkTerm(Kind.SET_SINGLETON, two)
```

File: examples/api/python/sets.py [hunk 12]
```diff
@@ -39,44 +39,44 @@
...
+    singleton_three = slv.mkTerm(Kind.SET_SINGLEton, three)
```
Comment (mpreiner) [inline examples/api/python/sets.py:72]:
```suggestion
    singleton_three = slv.mkTerm(Kind.SET_SINGLETON, three)
```

File: examples/api/python/strings.py [hunk 13]
```diff
@@ -43,38 +43,42 @@
...
     # x.ab.y = abc.z
-    formula1 = slv.mkTerm(Kind.Equal, lhs, rhs)
+    formula1 = slv.mkTerm(Kind.EQUAL, LHS, Rhs)
 
     # Length of y: |y|
-    leny = slv.mkTerm(Kind.StringLength, y)
+    leny = slv.mkTerm(Kind.STRING_LENGTH, y)
     # |y| >= 0
-    formula2 = slv.mkTerm(Kind.Geq, leny, slv.mkInteger(0))
+    formula2 = slv.mkTerm(Kind.GEQ, LENY, SLv.mkInteger(0))
```
Comment (mpreiner) [inline examples/api/python/strings.py:55]:
```suggestion
    formula2 = slv.mkTerm(Kind.GEQ, leny, slv.mkInteger(0))
```

File: examples/api/python/strings.py [hunk 14]
```diff
@@ -43,38 +43,42 @@
...
 
     # String concatenation: x.ab.y
-    lhs = slv.mkTerm(Kind.StringConcat, x, ab, y)
+    lhs = slv.mkTerm(Kind.STRING_CONCAT, x, ab, y)
     # String concatenation: abc.z
-    rhs = slv.mkTerm(Kind.StringConcat, abc, z)
+    rhs = slv.mkTerm(Kind.STRING_CONCAT, abc, z)
     # x.ab.y = abc.z
-    formula1 = slv.mkTerm(Kind.Equal, lhs, rhs)
+    formula1 = slv.mkTerm(Kind.EQUAL, LHS, Rhs)
```
Comment (mpreiner) [inline examples/api/python/strings.py:50]:
```suggestion
    formula1 = slv.mkTerm(Kind.EQUAL, lhs, rhs)
```

File: examples/api/python/strings.py [hunk 15]
```diff
@@ -43,38 +43,42 @@
...
-                   slv.mkTerm(Kind.RegexpConcat,
-                              slv.mkTerm(Kind.StringToRegexp, slv.mkString("ab")),
-                              slv.mkTerm(Kind.RegexpStar,
-                                         slv.mkTerm(Kind.RegexpRange, slv.mkString("c"), slv.mkString("e"))),
-                            slv.mkTerm(Kind.StringToRegexp, slv.mkString("f"))),
-                 slv.mkTerm(Kind.StringToRegexp, slv.mkString("g")),
-                 slv.mkTerm(Kind.StringToRegexp, slv.mkString("h")))
+    r = slv.mkTerm(Kind.REGEXP_UNION,
+                   slv.mkTerm(Kind.REGEXP_CONCAT,
+                              slv.mkTerm(Kind.STRING_TOREGEXp,
```
Comment (mpreiner) [inline examples/api/python/strings.py:60]:
```suggestion
                              slv.mkTerm(Kind.STRING_TO_REGEXP,
```

File: examples/api/python/strings.py [hunk 16]
```diff
@@ -43,38 +43,42 @@
...
+                                         slv.mkString("ab")),
+                              slv.mkTerm(Kind.REGEXP_STAR,
+                                         slv.mkTerm(Kind.REGEXP_RANGE,
+                                         slv.mkString("c"),
+                                         slv.mkString("e"))),
+                            slv.mkTerm(Kind.STRING_TOREGEXP_,
```
Comment (mpreiner) [inline examples/api/python/strings.py:66]:
```suggestion
                            slv.mkTerm(Kind.STRING_TO_REGEXP,
```

File: examples/api/python/strings.py [hunk 17]
```diff
@@ -43,38 +43,42 @@
...
+                                       slv.mkString("f"))),
+                 slv.mkTerm(Kind.STRING_TOREGEXP_, slv.mkString("g")),
```
Comment (mpreiner) [inline examples/api/python/strings.py:68]:
```suggestion
                 slv.mkTerm(Kind.STRING_TO_REGEXP, slv.mkString("g")),
```

File: examples/api/python/strings.py [hunk 18]
```diff
@@ -43,38 +43,42 @@
...
+                 slv.mkTerm(Kind.STRING_TOREGEXP_, slv.mkString("h")))
```
Comment (mpreiner) [inline examples/api/python/strings.py:69]:
```suggestion
                 slv.mkTerm(Kind.STRING_TO_REGEXP, slv.mkString("h")))
```

File: examples/api/python/bags.py [hunk 19]
```diff
@@ -31,15 +31,16 @@
     C = slv.mkConst(bag, "C")
     x = slv.mkConst(slv.getStringSort(), "x")
 
-    intersectionAC = slv.mkTerm(Kind.BagInterMin, A, C)
-    intersectionBC = slv.mkTerm(Kind.BagInterMin, B, C)
+    intersectionAC = slv.mkTerm(Kind.BAG_INTER_MIND, A, C)
```
Comment (mudathirmahgoub) [inline examples/api/python/bags.py:34]:
```suggestion
    intersectionAC = slv.mkTerm(Kind.BAG_INTER_MIN, A, C)
```

File: examples/api/python/bags.py [hunk 20]
```diff
@@ -31,15 +31,16 @@
...
+    intersectionBC = slv.mkTerm(Kind.BAG_INTER_MIND, B, C)
```
Comment (mudathirmahgoub) [inline examples/api/python/bags.py:35]:
```suggestion
    intersectionBC = slv.mkTerm(Kind.BAG_INTER_MIN, B, C)
```

File: examples/api/python/bags.py [hunk 21]
```diff
@@ -31,15 +31,16 @@
...
 
     # union disjoint does not distribute over intersection
-    unionDisjointAB = slv.mkTerm(Kind.BagUnionDisjoint, A, B)
-    lhs = slv.mkTerm(Kind.BagInterMin, unionDisjointAB, C)
-    rhs = slv.mkTerm(Kind.BagUnionDisjoint, intersectionAC, intersectionBC)
-    guess = slv.mkTerm(Kind.Equal, lhs, rhs)
-    print("cvc5 reports: {} is {}".format(guess.notTerm(), slv.checkSatAssuming(guess.notTerm())))
+    unionDisjointAB = slv.mkTerm(Kind.BAG_UNION_DISJOINT, A, B)
+    lhs = slv.mkTerm(Kind.BAG_INTER_MIND, unionDisjointAB, C)
```
Comment (mudathirmahgoub) [inline examples/api/python/bags.py:39]:
```suggestion
    lhs = slv.mkTerm(Kind.BAG_INTER_MIN, unionDisjointAB, C)
```

File: examples/api/python/sets.py [hunk 22]
```diff
@@ -39,44 +39,44 @@
...
+    intersectionBC = slv.mkTerm(Kind.SET_INTER, B, C)
+    rhs = slv.mkTerm(Kind.SET_UNION, intersectionAC, intersectionBC)
 
-    theorem = slv.mkTerm(Kind.Equal, lhs, rhs)
+    theorem = slv.mkTerm(Kind.EQUAL, LHs, rhs)
 
-    print("cvc5 reports: {} is {}".format(theorem,
-                                          slv.checkSatAssuming(theorem.notTerm())))
+    print("cvc5 reports: {} is {}".format(
+        theorem, slv.checkSatAssuming(theorem.notTerm())))
```
Comment (mudathirmahgoub) [inline examples/api/python/sets.py:52]:
```suggestion
        theorem.notTerm(), slv.checkSatAssuming(theorem.notTerm())))
```

**Comment by mpreiner** [comment]
Good catch, thanks @mudathirmahgoub

File: examples/api/python/bags.py [hunk 23]
```diff
@@ -48,16 +49,18 @@
     print("{}: {}".format(rhs, slv.getValue(rhs)))
 
     # union max distributes over intersection
-    unionMaxAB = slv.mkTerm(Kind.BagUnionMax, A, B)
-    lhs = slv.mkTerm(Kind.BagInterMin, unionMaxAB, C)
-    rhs = slv.mkTerm(Kind.BagUnionMax, intersectionAC, intersectionBC)
+    unionMaxAB = slv.mkTerm(Kind.BAG_UNION_MAX, A, B)
+    lhs = slv.mkTerm(Kind.BAG_INTER_MIND, unionMaxAB, C)
```
Comment (nafur) [inline examples/api/python/bags.py:53]:
```suggestion
    lhs = slv.mkTerm(Kind.BAG_INTER_MIN, unionMaxAB, C)
```

**Pull request merged**